### PR TITLE
qemu.cfg.host-kernel: Update host_rhel name

### PR DESCRIPTION
**Pull request by spcui**
Currently, When install Win8..1 guest on RHEL.7.1 host, will
create the incorrect case list and install Win8..0 and Win8..1
guest now, cause RHEL.7.1 conflicts with Win8.1 configs,
so this is a workaround, using 7.u1 to fix it.

### CODE REVIEW

**Comment by spcui** [comment]
1161504

**Comment by ypu** [comment]
Hi @spcui  Seems this is not the only cfg that have the similar problem. Can you update them all? Thanks

**Comment by xiagao** [comment]
Acked-by: Xiaoling Gao <xiagao@redhat.com>

**Comment by CongSmile** [comment]
Hi @spcui 

Agree with @ypu comment.

Could you check all such problems ?

Thanks.

**Comment by spcui** [comment]
Hi @ypu @CongSmile , double checked, we have a separate config file for every RHEL host and guest, so for installing rhel guest cannot meet this issue, so I think don't need to update all config.

**Comment by ypu** [comment]
@spcui As we talked the host and guest cfg should not have same sub variants at all. So please update this pullreq,

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by CongSmile** [comment]
Also need update host-kernel-check.cfg .

**Comment by xutian** [comment]
Hi @spcui 
do we need to update guest cfg  ?

thanks,
@xutian

**Comment by spcui** [comment]
@suqinhuang @xutian @CongSmile updated now, could you help review again? thanks

**Comment by CongSmile** [comment]
Agree with @xutian , we should also update guest cfg.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by suqinhuang** [comment]
no need to update guest name as host and guest name are uniq now.

Regards

File: qemu/tests/cfg/timerdevice.cfg [hunk 1]
```diff
@@ -90,7 +90,7 @@
                             vcpu_socket = 2
         - newer_msrs_support_check:
             only Linux
-            no Host_RHEL.3,4,5 Host_RHEL.6.0
+            no Host_RHEL.m3,4,5 Host_RHEL.m6.u0
             no RHEL.3,4,5 RHEL.6.0,1,2,3
```
Comment (CongSmile) [inline qemu/tests/cfg/timerdevice.cfg]: Should be no Host_RHEL.m3,m4,m5 here?

**Comment by CongSmile** [comment]
And could you please also update the cdrom_test.cfg, which has been merged in tree just now?

Thanks.

File: qemu/cfg/host-kernel/host-kernel-check.cfg [hunk 2]
```diff
@@ -19,54 +19,54 @@ variants:
...
+            - m7:
                 host_kernel_ver_str += ".7"
                 variants:
-                    - 0:
+                    - u0:
                         host_kernel_ver_str += ".0"
                         requires_kernel = [">= 3.2.1-0.8", "< 3.10.0-124"]
-                    - 1:
+                    - u1:
                         host_kernel_ver_str += ".1"
```
Comment (CongSmile) [inline qemu/cfg/host-kernel/host-kernel-check.cfg]: Hi @spcui 

Just curious, could you explain why not update host_kernel_ver_str?

Thanks.

Comment (spcui) [inline qemu/cfg/host-kernel/host-kernel-check.cfg]: host_kernel_ver_str only for creating host_dict, so not need update.

Comment (CongSmile) [inline qemu/cfg/host-kernel/host-kernel-check.cfg]: Thanks @spcui .

Could you also help update qemu_cpu.cfg?
Others are fine for me.

Thanks.

**Comment by ypu** [comment]
Hi @spcui seems guest-hw.cfg under virt also need updated.

Please use 'grep Host_RHEL * -r' in your test tree to find out all places need updated.

**Comment by spcui** [comment]
related: https://github.com/autotest/virt-test/pull/2307

File: qemu/tests/cfg/cdrom_test.cfg [hunk 3]
```diff
@@ -80,7 +80,7 @@
         eject_cdrom_cmd = "eject %s"
         close_cdrom_cmd = "eject -t %s"
         mount_cdrom_cmd = "mount %s %s"
-        HOST_RHEL.5:
```
Comment (vipmike007) [inline qemu/tests/cfg/cdrom_test.cfg]: I think this line should be  Host_RHEL.m5

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by spcui** [comment]
https://github.com/avocado-framework/avocado-vt/pull/129

**Comment by ypu** [comment]
As the related patch is merged. Will merge this one too

Thanks @spcui @suqinhuang @CongSmile

**Pull request merged**